docs/actions-nodejs.md

```diff
@@ -43,7 +43,7 @@ and demonstrate how to bundle multiple JavaScript files and third party dependen
 
   The CLI automatically infers the type of the action by using the source file extension.
   For `.js` source files, the action runs by using a Node.js runtime. You may specify
-  the Node.js runtime to use by explicitly specifying the parameter `--kind nodejs:10`, `--kind nodejs:8`, or `--kind nodejs:6`
+  the Node.js runtime to use by explicitly specifying the parameter `--kind nodejs:12`, `--kind nodejs:10`, `--kind nodejs:8`, or `--kind nodejs:6`
```

Comment:
remove nodejs:6?

Reply:
I'm not sure about this. It is still available in the project (for back compat reasons atm). Until it's removed - which is another PR, I think we should keep it in the docs?

Reply:
Remove it from the docs now